dashboard/client/src/pages/job/index.tsx

```diff
@@ -124,6 +127,14 @@ const JobList = () => {
                 ),
               }}
             />
+            <Autocomplete
+              style={{ height: 35, width: 150 }}
```

Comment:
```suggestion
              style={{ height: 35, width: 150 }}
```

nit: use `sx` instead of style. In other places where we use style, we want to transition to `sx`.

Reply:
solved.